geoh5py/objects/surveys/direct_current.py

```diff
@@ -180,7 +183,7 @@ def potential_electrodes(self):
         return self
 
     @property
-    def metadata(self) -> dict | None:
+    def metadata(self):
```

Comment:
is not metadata always json? thus a dict?
(Consistent with the setter that is not typed anymore, but why?)

Reply:
Brought back typing